contracts/eden/include/events.hpp

```diff
@@ -30,6 +30,12 @@ namespace eden
    //
    //    election_event_end
 
+   struct migration_event
+   {
+      uint16_t index;
```

Comment:
Why uint16_t?  variant stores the index as a varint, and we're unlikely to exceed a single byte in the near future.

Reply:
Ah oops my bad changing to `uint8_t`

Reply:
`varuint32`. This is 1 byte for the near future, but expands as needed.